src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int fistSide;

```

Comment:
Typo in variable name 'fistSide'; it should be 'firstSide'. Names should be clear and free from typos to avoid confusion and potential bugs.